src_web/kamaho-shokusu/src/Policy/FeatureUsageSummaryPolicy.php

```
@@ -0,0 +1,42 @@
+
+use App\Domain\ValueObject\UserRole;
+use Authorization\IdentityInterface;
+
+/**
+ * 機能使用頻度ダッシュボード認可ポリシー
+ *
+ * システム管理者（i_admin === 3）のみアクセスを許可する。
+ */
+class FeatureUsageSummaryPolicy
+{
+    public function canIndex(?IdentityInterface $user, mixed $resource): bool
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$resource'. (undefined)

(UnusedFormalParameter)